lib/ast-converter.js

```diff
@@ -256,6 +256,26 @@ function fixExports(node, result, ast) {
 }
 
 /**
+ * Returns true if a given TSNode is a token
+ * @param  {TSNode} node the TSNode
+ * @returns {boolean}     is a token
+ */
+function isToken(node) {
+     /**
+     * Note: We treat JsxText like a token (TypeScript doesn't classify it as one),
```

Comment:
Interesting, and thanks for keeping an eye on the project @DanielRosenwasser 😄 

I will take another look at this once TypeScript 2.2 hits stable.